src/PaymentPart/Output/FpdfOutput/Template/QrBillFooter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput\Template;
+
+use Fpdf\Fpdf;
+
+class QrBillFooter extends Fpdf
```

Comment:
Is this a good approach for the user?

I suggest that in real life, a developer will create some `Fpdf` instance, build invoice contents and in the end might want to add the qr bill to it. If they need to use this special implementation of `QrBillFooter` I would see this as problematic.

Could we find a way so the developer can use any existing Fpdf instance and inject into the qr bill library?

Reply:
You're right, but we need the SetDash function that must be an instance of Fpdf because the variables are protected and not public. I really dont know how to implement this function.
If you have an idea you can help me on this.

Reply:
> If you have an idea you can help me on this.

Maybe let's just delete it?
[The qr bill style guide always uses normal lines](https://github.com/sprain/php-swiss-qr-bill/blob/master/docs/specs/style-guide-en.pdf). It does not have to be a dashed line.

Reply:
I will try to make it differently, if it work we can use it othewise we can delete, it is not the most important for me.